contracts/SmartLoan.sol

```diff
@@ -392,9 +429,9 @@ contract SmartLoan is OwnableUpgradeable, PriceAwareUpgradeable, ReentrancyGuard
     _;
     uint256 LTV = getLTV();
     if (msg.sender != owner()) {
-      require(LTV >= MIN_SELLOUT_LTV, "This operation would result in a loan with LTV lower than Minimal Sellout LTV which would put loan's owner in a risk of an unnecessarily high loss.");
+      if (LTV < MIN_SELLOUT_LTV) revert PostSelloutLtvTooLow();
     }
-    require(LTV < MAX_LTV, "This operation would not result in bringing the loan back to a solvent state.");
+    if (LTV > MAX_LTV) revert PostSelloutLoanInsolvent();
```

Comment:
It's different from the original one. We should be clear in the description if MAX_SOLVENCY is inclusive or not.

Reply:
Already tackled in https://github.com/ava-loan/avaloan/pull/23